packages/integration-sdk-http-client/src/util.ts

```diff
@@ -0,0 +1,19 @@
+export function getUnixTimeNow() {
+  return Date.now() / 1000;
+}
+
+export async function sleep(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+/**
+ * Returns true when we have a status code that is an error but not retryable.
+ * @param statusCode
+ * @returns boolean
+ */
+export function isNonRetryableError(statusCode: number): boolean {
+  if (statusCode !== 429 && statusCode >= 400) {
```

Comment:
It depends on the API, but I believe there are a number of status codes > 400 that would be worth retrying.

Reply:
Yeah, and some APIs use 503 instead of 429 for rate limit errors.  I pulled it out into a function so we could more easily abstract it later on, but didn't put a todo.  I think the range of retryable as well as the rate limit error need to be configurable, possibly as an optional parameter into the constructor?

For now, I'll add a TODO like I should have.